frontend/ui/widget/doublespinwidget.lua

```diff
@@ -169,7 +169,7 @@ function DoubleSpinWidget:update(numberpicker_left_value, numberpicker_right_val
     if self.default_values then
         table.insert(buttons, {
             {
-                text = self.default_text or T(_("Apply default values: %1 / %2"),
+                text = self.default_text or T(_("Spin to default values: %1 / %2"),
```

Comment:
"Spin to..." feels a bit odd - dunno if that works in English. (I guess you mean: reset the spin/number picker to these values)
We had some discussions about the wording/applying or not in #8490.

Reply:
Thanks too loong to read now, but I will read it.

Not needed for the doublespinwidge in this PR anymore. I can drop it, if we don't find an easy solution.

Anyway to call it  `Apply` means apply to the user. Why should then the `Apply` button be clickable, when the values have already been applied (as the user tapped on that button)? (Apparently they have not been applied, they just got preset). -> Maybe `Preset to` might be a good phrase?

Reply:
BTW The DSW in the hyphenation context does not use the default text.